coordinator/tests/services/validation/bundle-validator.spec.ts

```diff
@@ -446,6 +449,18 @@ describe("MedicationRequest consistency checks", () => {
     expect(validationErrors).toHaveLength(1)
   })
 
+  test("Should throw error when GMP-number is only number in Practitioner.identifier", () => {
+    const testReference: fhir.Identifier = createIdentifier("https://fhir.hl7.org.uk/Id/gmp-number", "G1234567")
```

Comment:
You can replace this with fhir.createIdentifier and remove the additional import